DogTales/src/components/physics.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <bave/app.hpp>
+#include <bave/graphics/sprite.hpp>
+
+namespace component {
+
+static constexpr glm::vec2 max_velocity{1000.f, 1000.f};
+static constexpr float tick_limit{0.03f}; //this should possibly go somewhere else and be globally accessible
```

Comment:
Globals in headers should not be `static`: that will introduce new storage for the global in each translation unit (cpp file that includes the header). Just `constexpr` is fine here, though if you want to be pedantic, `inline constexpr`.